common/reviews/api/node-library-build.api.ts

```diff
@@ -1,13 +1,9 @@
 export declare function addSuppression(str: string): void;
 
 class ApiExtractorTask extends GulpTask<IApiExtractorTaskConfig> {
-  // (undocumented)
```

Comment:
@dgaeta @nickpape-msft Any idea what's going on here?  I just pulled a clean master and rebuilt, and I get this warning:

```
[17:05:35] ==================[ Finished ]==================
Warning - [api-extractor] You have changed the Public API signature for this pro
ject.  Please overwrite '../common/reviews/api/node-library-build.api.ts' with a
 copy of 'temp/node-library-build.api.ts' and then request an API review. See th
e Git repository README.md for more info.
[17:05:36] Project @microsoft/node-library-build version: 2.2.0
```

Maybe it is a regression from PR #84.  But if so, how was Danny able to get a passing Travis PR build?